examples/OpenCyphal-Blink/OpenCyphal-Blink.ino

```diff
@@ -63,8 +63,8 @@ ArduinoMCP2515 mcp2515([]() { digitalWrite(MKRCAN_MCP2515_CS_PIN, LOW); },
                        nullptr);
 
 CyphalHeap<Node::DEFAULT_O1HEAP_SIZE> node_heap;
-Node node_hdl(node_heap.data(), node_heap.size());
-Publisher<Heartbeat_1_0<>> heartbeat_pub = node_hdl.create_publisher<Heartbeat_1_0<>>(Heartbeat_1_0<>::PORT_ID);
+Node node_hdl(node_heap.data(), node_heap.size(), micros);
+Publisher<Heartbeat_1_0<>> heartbeat_pub = node_hdl.create_publisher<Heartbeat_1_0<>>(Heartbeat_1_0<>::PORT_ID, CANARD_DEFAULT_TRANSFER_ID_TIMEOUT_USEC);
```

Comment:
The transfer-ID timeout is only meaningful for incoming transfers, not publishers. Here you should pick some value based on the application requirements; a good default is the minimum expected publication period (perhaps 1 second in this case).

Reply:
Done in https://github.com/107-systems/107-Arduino-Cyphal/pull/183/commits/a644b9e587080670f321d37fff2112f75d85b5fd .